### PR TITLE
Add matter cover platform

### PR DESCRIPTION
**Pull request by hidaris**
## Proposed change

add cover to matter platform, update some outdated information.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/90262
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by marcelveldt** [comment]
There are unrelated changes in your PR, please only add the addition of the cover platform

File: source/_integrations/matter.markdown [hunk 1]
```diff
@@ -29,8 +31,6 @@ Matter is [the new standard for home automation](https://en.wikipedia.org/wiki/M
 
 Matter devices are available using either WiFi based communication or [Thread](/integrations/thread/), both are supported by Home Assistant. Bluetooth is used for adding new devices to your Matter network.
 
-Home Assistant only supports control of Matter devices. Home Assistant is not a bridge itself and it cannot turn devices within Home Assistant into Matter compatible devices.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]: Why did you remove this ?

File: source/_integrations/matter.markdown [hunk 2]
```diff
@@ -53,7 +53,7 @@ _If you run Home Assistant Container, Home Assistant Core, or you don’t want t
 
 While the support for Matter is evolving, we will regularly update the Matter integration with new features or device support. Because it might be hard to track what's supported and what not, we list the current state here and try to update this information as often as possible.
 
-Platform support in Home Assistant is currently limited to switches, lights, and (binary) sensors. The light platform is limited to _on/off_ and _brightness_ control only, support for _color_ and _color temperature_ control will be added soon.
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown:56]: ah, yes this was indeed outdated, fine to change within this PR

File: source/_integrations/matter.markdown [hunk 3]
```diff
@@ -167,6 +167,11 @@ Once the initial firmware upgrade to Matter is complete, the device can also be
 - [Eve Door & Window on Amazon](https://amzn.to/3RIU6ml)
 - [Eve Motion on Amazon](https://amzn.to/3jDujiP)
 
+### Zemismart Roller Shade Motor (cover)
```
Comment (marcelveldt) [inline source/_integrations/matter.markdown]: this almost feels like a commercial for this hardware. Maybe we should just go for a device compatibility table in a next PR where this device is also mentioned. For now, either include (neutral) device specific instructions or leave out this text. Thanks!

**Review by marcelveldt — CHANGES_REQUESTED** [review]
see inline comments

File: source/_integrations/matter.markdown (hunk 1 above)
Comment (hidaris) [reply source/_integrations/matter.markdown]: Yes, I realized I shouldn't remove here, sorry

**Review by marcelveldt — APPROVED** [review]

File: source/_integrations/matter.markdown (hunk 3 above)
Comment (hidaris) [reply source/_integrations/matter.markdown]: I agree with removing the text first, although the intention was to add the device I used to test it, but it does look like an ad, as you say

**Review by MartinHjelmare — APPROVED** [review]
